corehq/apps/reports/static/reports/javascripts/daterangepicker.js

```diff
@@ -26,6 +26,28 @@ $(function() {
             ranges: ranges,
             separator: separator
         });
+
+        $('.daterangepicker_start_input').hide();
+        $('.daterangepicker_end_input').hide();
+
+        $('.ranges .applyBtn').hide();
+        $('.ranges .cancelBtn').hide();
+
+        //the custom range button is the 4th li, but has no other
+        //defining class or id to select it
+        $('.ranges ul li:nth-child(4)').click(function() {
+            $('.ranges .applyBtn').show();
+            $('.ranges .cancelBtn').show();
+        });
+
+        $('.ranges ul li:not(:nth-child(4))').click(function() {
+            $('.ranges .applyBtn').hide();
+            $('.ranges .cancelBtn').hide();
+        });
+
+        $('#filter_range').change(function() {
+            $('.ranges .applyBtn').click();
```

Comment:
triggering a click event here feels a little bit like a hack. I would trigger a specific event that is namespaced to our reports that we use to pick up on that interaction.